schema-registry/common/src/main/java/com/hortonworks/registries/schemaregistry/errors/ConfigTypeConversionException.java

```diff
@@ -0,0 +1,8 @@
+package com.hortonworks.registries.schemaregistry.errors;
```

Comment:
Can you add the Cloudera license header here?

Reply:
Yes, of course - I guess, it is the same as for other files.

Reply:
Yup, same as the other files.